networking/network_observability/network-observability-overview.adoc

```diff
@@ -36,6 +36,7 @@ The Network Observability Operator provides the Flow Collector API custom resour
 
 The {product-title} console offers the *Overview* tab which displays the overall aggregated metrics of the network traffic flow on the cluster. The information can be displayed by node, namespace, owner, pod, and service. Filters and display options can further refine the metrics.
 
+Additionally, in *Observe* -> *Dashboards*, the *Netobserv* dashboard provides a quick overview of the network flows in your {product-title} cluster. You can view metrics, such as *Top flow rates per source and destination namespaces*, *Top byte rates received per source and destination nodes*, and *Top byte rates received per source and destination workloads*. Also in *Observe* -> *Dashboards*, the *Netobserv/Health* dashboard provides metrics about the health of the Operator.
```

Comment:
Having _Additionally_ and _Also_ in the same paragraph sounds odd to me. Maybe it's just me.

Reply:
Agree, I had to rework this section with an unordered list instead of a comma-delimited list, and I separated out that "Also" sentence.